tests/cupy_tests/cuda_tests/test_stream.py

```diff
@@ -6,21 +6,32 @@
 from cupy.testing import attr
 
 
+@testing.parameterize(
+    *testing.product({
+        'stream': [cuda.Stream.null, cuda.Stream.ptds],
+    }))
 class TestStream(unittest.TestCase):
 
     @attr.gpu
     def test_eq(self):
-        null0 = cuda.Stream.null
-        null1 = cuda.Stream(True)
-        null2 = cuda.Stream(True)
-        null3 = cuda.Stream()
+        null0 = self.stream
+        if self.stream == cuda.Stream.null:
+            null1 = cuda.Stream(True)
+            null2 = cuda.Stream(True)
+            null3 = cuda.Stream(ptds=True)
+        else:
+            null1 = cuda.Stream(ptds=True)
+            null2 = cuda.Stream(ptds=True)
+            null3 = cuda.Stream(True)
+        null4 = cuda.Stream()
 
         assert null0 == null1
         assert null1 == null2
         assert null2 != null3
+        assert null2 != null4
 
-    def check_del(self, null):
-        stream = cuda.Stream(null=null).use()
+    def check_del(self, test_stream, null, ptds):
```

Comment:
I don't think `test_stream` is used?

Reply:
You're right, another leftover from my initial attempt at having this working. Should be fixed now. :)